source/fundamentals/authentication.txt

```diff
@@ -1 +1,208 @@
-.. _csharp-authentication-mechanisms:
\ No newline at end of file
+.. _csharp-authentication-mechanisms:
+
+=========================
+Authentication Mechanisms
+=========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to authenticate with MongoDB using each
+**authentication mechanism** available in the {+mongo-community+}.
+Authentication mechanisms are processes by which the driver and server confirm
+identity and establish trust to ensure security before connecting.
```

Comment:
The wording here is a little unclear. confirm whose identity? Establish trust between which entities?

Reply:
Reworded this to hopefully make it more clear